google/cloud/storage/client.h

```diff
@@ -1090,7 +1168,27 @@ class Client {
     request.set_multiple_options(std::forward<Options>(options)...);
     return raw_client_->PatchBucketAcl(request).second;
   }
+  //@}
 
+  //@{
+  /**
+   * @name Object Access Control List operations.
+   *
+   * You can control who has access to your GCS buckets and objects as
+   * well as what level of access they have.
+   *
+   * @note When you upload (create) an object the entity that uploads the object
+   *     is listed as the object owner. This can be a user or a service account,
+   *     depending on what credentials are used to authenticate with GCS.
+   *     Object ownership cannot be changed by modifying ACLs. You can change
+   *     object ownership only by overwriting the object.
```

Comment:
Nit: maybe clarify and change this to "by overwriting the object while authenticated as the desired owner."

Reply:
Done.